manifests/config.pp

```diff
@@ -54,6 +54,7 @@
   $init_limit = 10,
   $sync_limit = 5,
   $log4j_file = undef,
+  $notify_zookeeper = true,
```

Comment:
Don't we need to add this to init.pp as well? (also we are calling the 'class { 'zookeeper' in our module). 

Also, comments on documentation and variable naming (in other review) also apply.